libraries/app/include/graphene/app/database_api.hpp

```diff
@@ -268,6 +268,25 @@ class database_api
        */
       vector<optional<account_object>> get_accounts(const vector<account_id_type>& account_ids)const;
 
+      /**
+       * @brief Fetch all orders relevant to the specified accounts sorted descendingly by price
+       *
+       * @param name_or_id  Each item must be the name or ID of an account to retrieve
+       * @param base  Base asset
+       * @param quote  Quote asset
+       * @param limit  Max items to fetch
+       * @param start_id  Start order id, fetch orders which price lower than this order
```

Comment:
`lower` is not accurate. BTW if the price is the same, order id should be equal or higher?